src/components/Coins.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { coinList, coinFilter } from '../redux/coins/coinsSlice';
+import Coin from './Coin';
+
+const Coins = () => {
+  const dispatch = useDispatch();
+  const crytoList = useSelector((store) => store.cryto);
+  useEffect(() => {
+    dispatch(coinList());
+  });
+  const changeHandler = (e) => {
+    dispatch(coinFilter(e.target.value));
+  };
+
+  const searchHandler = (e) => {
+    e.preventDefault();
+    window.location.href = `/detail/${e.target.coinSearch.value}`;
+  };
+
+  return (
+    <section>
+      <div className="infos d-none">
+        <select className="filter" onChange={changeHandler}>
+          <option value="price"> Rank </option>
+          <option value="name"> Name </option>
+          <option value="vol"> Price </option>
+        </select>
+        <form onSubmit={searchHandler}>
+          <input type="text" name="coinSearch" placeholder="Crypto cryto name" required />
+          <input type="submit" value="Search" />
+        </form>
```

Comment:
- [ ] **Good job on implementing the suggested changes 👏 👏** however, As suggested by the previous reviewer, It would be better to add a filter or a search input for your page where a user can type the name like USDT, ETH, etc and get the filtered items on the home page 🙏 

   Currently, the search input is not visible on your page as a searchable input, Kindly fix this for the best user experience and to complete the project requirements 👍 👍 

![image](https://user-images.githubusercontent.com/102757126/225911950-40c66bb0-f051-43bb-bd46-59d2c03c6d2c.png)

   _I think this [youtube tutorial](https://www.youtube.com/watch?v=ZoayCCDHFiI) can help 😎😎😎_